content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md

```diff
@@ -8,22 +8,44 @@ aliases:
 keywords: [platform-setup,kubernetes,minikube]
 ---
 
-Follow these instructions to prepare Minikube for Istio.
+Follow these instructions to prepare Minikube for Istio installation with sufficient
+resources to run Istio and some basic applications.
 
 1. To run Istio locally, install the latest version of
    [Minikube](https://kubernetes.io/docs/setup/minikube/), version **0.28.1 or
    later**.
 
-1. Select a
-   [VM driver](https://kubernetes.io/docs/setup/minikube/#quickstart)
-   and substitute `your_vm_driver_choice` below with the installed virtual
-   machine (VM) driver. To install Istio control plane components and addons,
-   as well as other applications,
-   we recommend starting Minikube with 8192 `MB` of memory and 4 `CPUs`:
+1. Select a [Minikube VM driver](https://kubernetes.io/docs/setup/minikube/#quickstart)
+   and install it using the driver installation techniques.  Finally export the
+   driver to the environment.  An example using kvm2 driver is:
 
-    On Kubernetes **1.13**:
+    {{< text bash >}}
```

Comment:
Another option is to run `minikube config set vm-driver kvm2` and not have to add to your profile.